### ISSUE TITLE
Out of memory

### ISSUE THREAD

**Issue by manmanjun** [layumi/Seg-Uncertainty#9]
I use two titanx gpus.Can you tell me the type of GPU you use？
![image](https://user-images.githubusercontent.com/41903038/86942881-4cfb5f00-c178-11ea-8a5c-ffa8a3c3d074.png)

**Comment by layumi** [comment]
Hi @manmanjun 
I mainly use the 3x P5000 with 16GB memory. For the stage-II, the input size is small. I use the 3x 2080TI with 11GB.

**Comment by manmanjun** [comment]
Hi, in the paper, Rectifying Pseudo Label Learning via Uncertainty Estimation for Domain Adaptive Semantic Segmentation, I have a question about the last picture: Prediction Confidence in Fig.5. What is the way to get confidence ?

**Comment by layumi** [comment]
Hi  @manmanjun 
It is the prediction score after softmax function.

**Comment by KaiiZhang** [comment]
Hi, @layumi 
Thanks for your code! I use four 2080Ti gpus, which memory is only 11GB. So in the Stage-I, i may have to set crop-size to 512,256(It is being trained now). I am curious whether high-resolution training images are critical at this stage？

**Comment by layumi** [comment]
@Zachzhang97 
Yes. For the first stage, the resolution of the input image is really important.

**Comment by KaiiZhang** [comment]
Sounds sad……Do you have any suggestions for reproducing your project on the GPU with smaller memory？

**Comment by manmanjun** [comment]
> Sounds sad……Do you have any suggestions for reproducing your project on the GPU with smaller memory？

 Apex is all you need.

**Comment by layumi** [comment]
@Zachzhang97 
Yes. You may try float16, which will affect the performance but is way better than the small input resolution.

**Comment by KaiiZhang** [comment]
> @Zachzhang97
> Yes. You may try float16, which will affect the performance but is way better than the small input resolution.

All right, thanks a lot.

**Comment by KaiiZhang** [comment]
> > Sounds sad……Do you have any suggestions for reproducing your project on the GPU with smaller memory？
> 
> Apex is all you need.

 Sorry，i don't understand what you mean.

**Comment by KaiiZhang** [comment]
> > Sounds sad……Do you have any suggestions for reproducing your project on the GPU with smaller memory？
> 
> Apex is all you need.

Oh, I understand what you mean, thanks !
